.travis.yml

```diff
@@ -13,7 +13,7 @@ bundler_args: "--without development --path=~/.bundle"
 before_script:
 - cp config/database.yml.travis config/database.yml
 - psql -c 'create database travis_ci_test;' -U postgres
-- bundle exec rake db:schema:load db:migrate RAILS_ENV=test
+- psql -U postgres -q -d travis_ci_test -f db/structure.sql
```

Comment:
I was trying to see if I could shave some time off the build by not loading rake at all, but it's currently a difference of maybe 5 seconds.